google/auth/crypt.py

```diff
@@ -206,7 +206,59 @@ def sign(self, message):
         raise NotImplementedError('Sign must be implemented')
 
 
-class RSASigner(Signer):
+class _FromServiceAccountMixin(object):
```

Comment:
@jonparrott I wanted to re-use these factories, as they truly only depend on `from_string`, but I wasn't sure how to go about it (i.e. the mixin is a little gross). WDYT?